src/components/activity/FormInput.js

```diff
@@ -0,0 +1,59 @@
+import {useState} from 'react';
+import {FormStyles} from './styles/FormInput.styles';
+import {nanoid} from 'nanoid';
+
+function FormInput({addActivity}) {
+  const [activity, setActivity] = useState({
+    id: nanoid(),
+    activity: '',
+    startTime: '00:00',
+    endTime: '00:00',
+  });
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    console.log(event);
```

Comment:
Please delete this console log. There should be no console logs in your final PR.